.cirrus.yml

```diff
@@ -201,8 +201,11 @@ task:
         - flutter channel $CHANNEL
         - flutter upgrade
         - ./script/incremental_build.sh build-examples --ipa
-        - ./script/incremental_build.sh drive-examples --ios
         - ./script/incremental_build.sh xctest --target RunnerUITests --skip $PLUGINS_TO_SKIP_XCTESTS --ios-destination "platform=iOS Simulator,name=iPhone 11,OS=14.3"
+        # `drive-examples` contains integration tests, which changes the UI of the application.
+        # This UI change sometimes affects `xctest`.
+        # So we run `drive-examples` after `xctest`, changing the order will result ci failure.
+        - ./script/incremental_build.sh drive-examples --ios
```

Comment:
@stuartmorgan This is required to successfully run the XCUITests.
We never discovered this because we never had a plugin runs both integration tests and XCUITests. PTAL

Reply:
LG